app/reports/page-content.tsx

```diff
@@ -126,6 +127,12 @@ const ResolvedFilters = () => {
           onClick: () => updateParams('includeMuted', true),
           isActive: includeMuted === 'true',
         },
+        {
+          id: 'onlyMuted',
+          text: 'Only Muted',
```

Comment:
What results should you get when "only muted" is on?

Reply:
Only subjects that were muted or reporters that were muted will show up in the queue. This is to provide a quick way to glance through all mutes in case manual reverse/check is necessary.